policy/modules/contrib/rtorrent.te

```diff
@@ -0,0 +1,95 @@
+policy_module(rtorrent, 1.0.1)
+
+########################################
+#
+# Declarations
+#
+## <desc>
+## <p>
+## Allow rtorrent to use send mails
+## </p>
+## </desc>
+gen_tunable(rtorrent_send_mails, false)
+
+## <desc>
+## <p>
+## Enable necessary permissions for rutorrent
+## </p>
+## </desc>
+gen_tunable(rtorrent_enable_rutorrent, false)
+
+attribute rtorrentdomain;
+
+attribute_role rtorrent_roles;
+roleattribute system_r rtorrent_roles;
+
+type rtorrent_t;
+type rtorrent_exec_t;
+userdom_user_application_domain(rtorrent_t, rtorrent_exec_t)
+role rtorrent_roles types rtorrent_t;
+
+########################################
+#
+# rtorrent local policy
+#
+
+type rtorrent_port_t;
+corenet_port(rtorrent_port_t)
+allow rtorrent_t rtorrent_port_t:tcp_socket name_bind;
+
+userdom_read_user_home_content_symlinks(rtorrent_t)
+
+allow rtorrent_t self:process setpgid;
+allow rtorrent_t self:netlink_audit_socket { create_socket_perms nlmsg_relay };
+allow rtorrent_t self:fifo_file rw_fifo_file_perms;
+allow rtorrent_t self:tcp_socket create_stream_socket_perms;
+allow rtorrent_t self:unix_stream_socket connectto;
+
+allow rtorrent_t self:netlink_route_socket { bind create nlmsg_read };
+allow rtorrent_t self:udp_socket { connect create getattr };
+nscd_shm_use(rtorrent_t)
```

Comment:
These above 3 lines are provided with the more suitable `auth_use_nsswitch()` interface. Call that instead.

Reply:
O, nevermind you called it below. So these 3 lines are redundant and can be removed.